### PR TITLE
Converts Suit Storage Units to use radials!

### PR DESCRIPTION
**Pull request by Tsar-Salat**
Ports:
- https://github.com/tgstation/tgstation/pull/53354

## About The Pull Request

Converts suit storage units to use radials instead of ugly list inputs

dreamseeker_Fvq2lWm07S

## Why It's Good For The Game

Modsuit port atomization

Looks a lot cleaner, less ugly white boxes on the screen the better

Basic things that you can only put a really limited amount of items in should use Radials. It is a complete waste to put them into lists. 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/8fd8e0f5-7868-4d3e-8a4d-1f0ad2998627

</details>

## Changelog
:cl: rkz, Mothblocks
add: Suit Storage Units now use radials!
add: All engineer Suit Storage Units now start with magboots in them.
rscadd: SSU radials
code: convert SSU code to update_appearance
/:cl:

### CODE REVIEW

**Review by itsmeow — APPROVED** [review]
yesyeyesyesyesyseyseyes

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -472,86 +574,3 @@
...
 		open_machine()
-
-
-/obj/machinery/suit_storage_unit/ui_state(mob/user)
-	return GLOB.notcontained_state
-
-/obj/machinery/suit_storage_unit/ui_interact(mob/user, datum/tgui/ui)
-	ui = SStgui.try_update_ui(user, src, ui)
-	if(!ui)
-		ui = new(user, src, "SuitStorageUnit")
```
Comment (itsmeow) [inline code/game/machinery/suit_storage_unit.dm:483]: remove the SuitStorageUnit file

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by itsmeow** [comment]
Do magboots still fit in it?

**Comment by Tsar-Salat** [comment]
> Do magboots still fit in it?

image

Yes.

I also took the liberty of updating the file to update_appearance standards, and removing all the ui_update calls. 

It doesnt use TGUI anymore, so those are unecessary.

File: code/game/machinery/suit_storage_unit.dm (hunk 1 above)
Comment (Tsar-Salat) [reply code/game/machinery/suit_storage_unit.dm:483]: removed in 648bc2c

**Review by itsmeow — APPROVED** [review]
Looks good to me. Thanks for the update_appearance refactor.

**Review by PestoVerde322 — APPROVED** [review]
It good.

**Pull request merged**